systemtests/src/test/java/io/enmasse/systemtest/sharedinfra/MessagingAddressTest.java

```diff
@@ -57,7 +51,8 @@
 
     private MessagingTenant tenant;
     private MessagingEndpoint endpoint;
-
+    private MessagingClientRunner clientRunner = new MessagingClientRunner();
```

Comment:
Maybe rename this to MessagingClientManager and put it in the TestBase? (CC @kornys ). This should be setup and teardown for each test method.

Reply:
I am kind of scary to name it manager because that would way to start thinking about it as manager and I really don't know if I want to face situation that we have singleton manager for everything. But I am open to discussion, naming is hard :D

Reply:
Yeah, I don't like the manager name maybe, but I think it should be something in the test base similar to the resource manager, that is properly cleaned up between tests etc.

Reply:
I agree with testBase, there is a space for this cleaning in before each/after each. Anyway please keep testBase clean :) I don't want to paste here common methods or asserts methods etc... :)

Reply:
Are you saying assertSendMessageForServiceBrokerAddress does not belong there @kornys ? :rofl:

Reply:
I am thinking that if we do it right we don't need Manager for clients because cleaning is usually not hard, it's just shutdown for executorService, so I would rather stay with this layer and see how it's working. I think that there will be more useful resources that would need that wdyt @lulf ?

Reply:
@obabec Call it runner or whatever, but we need something per test that is maintaining the list of clients that got created, and 'clears' the list between test methods.